test/testenv/appframework_utils.go

```diff
@@ -311,3 +328,162 @@ func WaitforPhaseChange(deployment *Deployment, testenvInstance *TestEnv, name s
 		time.Sleep(1 * time.Second)
 	}
 }
+
+// Verifications will perform several verifications needed between the different steps of App Framework tests
+func Verifications(deployment *Deployment, testenvInstance *TestEnv, appSource []AppSourceInfo, appVersion string, splunkPodAge map[string]time.Time, status string, clusterManagerBundleHash string, scaling string) string {
+	/* Function Steps
+	 * Verify apps 'download' state for all CRs
+	 * Verify apps 'podCopy' state for all CRs
+	 * Verify apps packages are deleted from the operator pod for all CRs
+	 * Verify apps 'install' state for all CRs
+	 * Verify apps packages are deleted from the CR pods
+	 * Verify bundle push is successful
+	 * Verify apps are copied to correct location on CR pods
+	 * Verify no pods did reset
+	 * Verify apps are installed to correct location on CR pods
+	 */
+	// Verify apps 'Download' and 'PodCopy' states for all CRs
+	phases := []enterpriseApi.AppPhaseType{enterpriseApi.PhaseDownload, enterpriseApi.PhasePodCopy}
+	for _, phase := range phases {
```

Comment:
Can be replace with `for _, phase := range []enterpriseApi.AppPhaseType{enterpriseApi.PhaseDownload, enterpriseApi.PhasePodCopy} {`